### DISCUSSION TITLE
LS rebooting of it's own accord

### DISCUSSION THREAD

**Discussion by SpeedyCharly** [AzuraCast/AzuraCast#4926] [Q&A] [answered]
[x-mas morn self restart.txt](https://github.com/AzuraCast/AzuraCast/files/7776480/x-mas.morn.self.restart.txt)

6:37 am - everyone in bed... why would LS restart of it's own accord?

'Splain that to me...

**Comment by BusterNeece** [comment, marked as answer]
I saw this in a few of the other log reports. I think maybe the Liquidsoap issue was closed because the primary concern (the timeouts) was resolved, but the secondary issue of occasional restarts was not.

Could you create another issue over there specifically for the restarts? 

I realize this upgrade has been a little taxing on everyone from the station operators to me and the LS team, but we're getting it figured out bit by bit.
